spec/board_spec.rb

```diff
@@ -0,0 +1,99 @@
+require './lib/board'
+
+describe Board do
+  let(:gameboard) { Board.new }
+  describe '#play_again' do
+    it 'should output an array of 9 dashes' do
+      expect(gameboard.play_again).to eq(%w[- - - - - - - - -])
+    end
+  end
+
+  describe '#horizontal_winner?(player)' do
+    it 'should return true when get a horizontal line of symbol X' do
```

Comment:
* [ ] Currently, the test for horizontal winner captures only one situation thus ```1,2,3``` very well, but is silent on position ```4,5,6``` and ```7,8,9```. Kindly add the situation whereby the winner is at position ```4,5,6``` or ```7,8,9``` to match the implementation of the method.